### ISSUE TITLE
Refine search result cards

### ISSUE THREAD

**Issue by rreitmann** [dzhw/metadatamanagement#687]
- [x] remove data set titles (TODOS) from variable search result card
- [x] remove TODO from question search result card (e.g. "Fragebogen se19")
- [x] tab order should be: studies, questions, variables, surveys, datasets, publications
- [x] put hardcoded title and abstract into publication search result

**Comment by rreitmann** [comment]
@MaxDerPraktikant Please test the search result cards in all browsers...
